msteams-platform/toolkit/teams-toolkit-overview-visual-studio.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Teams Toolkit overview for Visual Studio
+author: surbhigupta
+description: In this module, learn Teams Toolkit Overview for Visual Studio
+ms.author: v-amprasad
+ms.localizationpriority: medium
+ms.topic: overview
+ms.date: 05/24/2022
+---
+
+# Teams Toolkit overview for Visual Studio
+
```

Comment:
I am wondering is this a Zone Pivot in the same Overview page with VS Code right?

Reply:
We are not able to understand this comment, could you please clarify?